src/main/java/org/candlepin/subscriptions/tally/facts/FactNormalizer.java

```diff
@@ -108,7 +108,7 @@ private void normalizeUnits(NormalizedFacts normalizedFacts, InventoryHostFacts
     }
 
     private boolean isVirtual(InventoryHostFacts hostFacts) {
-        return hostFacts.isVirtual() ||
+        return hostFacts.isVirtual() || !StringUtils.isEmpty(hostFacts.getSatelliteHypervisorUuid()) ||
```

Comment:
The `hasText` method makes it so you don't have to have negation in there.  I think it reads easier. 

```suggestion
        return hostFacts.isVirtual() || StringUtils.hasText(hostFacts.getSatelliteHypervisorUuid()) ||
```

Reply:
Done!